xgi/drawing/xgi_pylab.py

```diff
@@ -54,29 +56,45 @@ def draw(
     ax : matplotlib.pyplot.axes (default=None)
 
     edge_lc : color (str, dict, or iterable, default='black')
-    Color of the edges (dyadic links and borders of the hyperedges).  If str, use the
-    same color for all edges.  If a dict, must contain (edge_id: color_str) pairs.  If
-    iterable, assume the colors are specified in the same order as the edges are found
-    in H.edges.
+        Color of the edges (dyadic links and borders of the hyperedges).  If str, use the
```

Comment:
I get the naming of edge_lc, edge_lw, and edge_fc, but at first I found it a bit confusing. Maybe we should at least explicitly expand and refer to the abbrevations (lc, fc, lw) in the docsting? Maybe also allow for a long version of these arguments?
Ideas that I don't think are great (but can't find betterones):
"(Line) color of the edges" or "Color of the (line) edges". 

Or change the names?
edge_dyadic_c, edge_dyadic_lw, edge_c
or
edge_c, edge_lw, hyperedge_c

Reply:
Also, does edge_lc actually color "borders of the hyperedges"? It's not so obvious from the viz examples. Or hyperedges have a much thiner lw that is not argument.

Reply:
Good question. I think I'm in favor of edge_lc only applying to dyadic links. And in the reference to the names, what about changing `edge_lc` to `linkcolor` or `dyadcolor`, `edge_lw` to `edge_linewidth`, etc. and actually expand the names? It's a balance between readability and brevity and I'm not sure what is best.

Reply:
I agree with edge_lc only applying to dyadic links. I checked in the code, right now `edge_lc` is also used for the border of hyperedges, but with fixed small width of 0.5. To draw hypergraphs, I would maybe replace and use `edge_fc` instead. (We could even have an extra argument specifically for that.. 😛 )

Re. naming, yea why not, then
- `dyad_color`and `dyad_lw` for dyads
- `edge_fc` (and potentially `edge_lc`) for other hyperedges
It would introduce the word dyad to XGI, which might not be the best, but I don't have a better idea to distinguish pairwise edges and others. 

Re. expanding, I'm not sure what is best either. Maybe keep the abbreviated version for the actual argument, but expand in the docstring explanation:
- `edge_fc : Face color of the edges (of more than 2 nodes)` 

(matplotlib allows both lw and linewidth as argument, we could have the same?)

Reply:
Done. Good point.